content/blog/2016/2016-08-25-sauce-pipeline.adoc

```diff
@@ -0,0 +1,262 @@
+---
+layout: post
+title: "Running Sauce On-Demand in Pipeline"
+tags:
+- pipeline
+- plugins
+- saucelabs
+- selenium
+- nightwatch
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at Cloudbees.
+
+The link:https://wiki.jenkins-ci.org/display/JENKINS/Sauce+OnDemand+Plugin[Sauce OnDemand Plugin]
+let's your Jenkins pipelines use Sauce Labs to run your Selenium tests on multiple browsers and platforms.
+
+=== Pipeline Hearts UI Testing, Too
+There comes a time in every project where you'll feel it's time checkin your build pipeline as code.
+I've been working with CI/CD and Jenkins for while - my tolerance for pipeline outside of code is extremely low.
+I'm a developer, not an operator.
+I may do some early prototyping in a Freestyle job via the Jenkins UI, but once I have even the simplest case working I'll refactor that to Pipeline code.
+
+I recently started looking for a way to do browser UI testing for an open-source javascript project I maintain.
```

Comment:
s/javascript/JavaScript/

I'm not sure on the grammer at the end of the sentence, I think there is a preposition missing? E.g. "an open source JavaScript which I maintain."

This is something @omehegan is much better at me at though.